README.md

```diff
@@ -87,6 +87,24 @@ Would you rather dive into the code? Then you are already in the right place!
 
 ---
 
+# Quick look at the high-level architecture
+
+![Mender architecture](mender_architecture.png)
```

Comment:
Is the original hosted in a Google document somewhere that you can link to with `<!-- LINK -->`?

Reply:
@kacf we can do that, but Google Drive is not very reliable as people can remove or modify documents and we have zero protection from this.

Reply:
Of course, it's not an absolute source. But better than having nothing.

Reply:
@kacf so to be clear. You want me to add `<!-- https://drive.google.com/file/d/1pKfJ-eRHYrDYZW7jCTHI7_D9tEF-rKDz -->` under the image as a comment?

Reply:
Yes, so that if it needs editing later, it can be found again. There are already several of these spread around the repository, like [the state diagrams](https://drive.google.com/drive/folders/0B6EXyILUYX54TWR5THZVbUZNR3M?resourcekey=0-thOspORP_ySJ0DnGuWvNRw) for example.